readinglist/views/article.py

```diff
@@ -14,14 +14,29 @@ def preparer(self, appstruct):
         return strip_whitespace(appstruct)
 
 
+class URL(SchemaNode):
+    """String representing a URL."""
+    schema_type = String
+    validator = colander.All(colander.url, colander.Length(min=1, max=2048))
```

Comment:
open for discussion: should we truncate the URL when it's longer than what we want? (rather than rejecting it)